ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -80,6 +80,10 @@ public extension Colors {
         public static var background = UIColor(light: surfacePrimary, dark: surfaceSecondary)
         public static var popoverBackground = UIColor(light: surfacePrimary, dark: surfaceQuaternary)
     }
+
+    // Objective-C support
+    @objc static var drawerBackground: UIColor { return Drawer.background }
+    @objc static var popoverBackground: UIColor { return Drawer.popoverBackground }
```

Comment:
Thoughts on just moving this out of a Colors extension (where it has to be in a Drawer struct) and into Drawer so we don't need this dance to make it consumable by objC?

Reply:
Also do we need to specify public? Or does @objc do that implicitly?

Reply:
> Thoughts on just moving this out of a Colors extension (where it has to be in a Drawer struct) and into Drawer so we don't need this dance to make it consumable by objC?

Yeah, I thought w were going for that for future changes, but I guess we can do that next time 🙂